cuFOLIO_portfolio_optimization/CVaR/02_backtesting.ipynb

```diff
@@ -29,6 +29,24 @@
     "Before diving into portfolio optimization and backtesting, we need to import the necessary libraries."
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "758fb7e3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Install dependencies\n",
+    "# This cell only needs to be run once, typically after setting up the environment.\n",
+    "# If dependencies are already installed, you can comment out or skip this cell.\n",
+    "\n",
+    "# Install cuOpt (if not already installed)\n",
+    "# !pip install --upgrade --user --extra-index-url https://pypi.nvidia.com -q cuopt-cu12==25.10.* \n",
```

Comment:
same here with 25.10. Do we really want to hardcode it?

Reply:
Nah, we don't need to, I just left what it was in other notebook. I think you were working on a PR to fox those.